app/shared/ui-wizard/src/commonMain/kotlin/ui/wizard/step/ConfigureProxy.kt

```diff
@@ -0,0 +1,376 @@
+        actions = {
+            TextButton({ onUpdate(currentConfig.value) }) {
+                Text("保存并测试")
+            }
+        },
+        useThinHeader = true,
+        modifier = modifier,
+    ) {
+        ProxyMode.entries.forEach { mode ->
+            val interactionSource = remember { MutableInteractionSource() }
+            val updateCurrentMode = {
+                currentConfig.update(
+                    currentConfig.value.copy(currentConfig.value.default.copy(mode = mode)),
+                )
+            }
+            TextItem(
+                title = { Text(renderProxyConfigModeName(mode)) },
+                icon = {
+                    RadioButton(
+                        selected = currentConfig.value.default.mode == mode,
+                        onClick = updateCurrentMode,
+                        interactionSource = interactionSource,
+                    )
+                },
+                modifier = Modifier
+                    .fillMaxWidth()
+                    .clickable(
+                        interactionSource = interactionSource,
+                        indication = LocalIndication.current,
+                        onClick = updateCurrentMode,
+                    ),
+            )
+            AnimatedVisibility(visible = currentConfig.value.default.mode == mode) {
+                Column {
+                    when (mode) {
+                        ProxyMode.DISABLED -> {}
+                        ProxyMode.SYSTEM -> {
+                            SystemProxyConfig(systemProxy)
+                        }
+
+                        ProxyMode.CUSTOM -> {
+                            CustomProxyConfig(currentConfig.value, currentConfig)
+                        }
+                    }
+                }
+            }
+        }
+    }
+}
+
+
+@Composable
+internal fun ConfigureProxy(
+    config: ProxySettings,
+    testRunning: Boolean,
+    systemProxy: SystemProxyPresentation,
+    testItems: List<ProxyTestItem>,
+    onUpdate: (config: ProxySettings) -> Unit,
+    onRequestReTest: () -> Unit,
+    modifier: Modifier = Modifier,
+    layoutParams: WizardLayoutParams = WizardLayoutParams.Default
+) {
+    var editingProxy by rememberSaveable { mutableStateOf(false) }
+
+    SettingsTab(modifier = modifier) {
+        Column {
+            ProxyTestStatusGroup(
+                testRunning,
+                config.default.mode,
+                testItems,
+                onRequestReTest = onRequestReTest,
+            ) {
+                testItems.forEach { item ->
+                    ProxyTestItemView(item, modifier = Modifier)
+                }
+            }
+
+            AnimatedContent(
+                editingProxy,
+                modifier = Modifier.animateContentSize(),
+                transitionSpec = {
+                    fadeIn(animationSpec = tween(220, delayMillis = 90))
+                        .togetherWith(fadeOut(animationSpec = tween(90)))
+                },
```

Comment:
Didn't follow clean fade. 220 and 90 are "random" numbers.

Use the predefined transitions form AniThemeDefaults: `AniThemeDefaults.standardAnimatedContentTransition` .